### PR TITLE
[FIX] avoid error on creating runbot budget if repo runbot still does…

### PR DESCRIPTION
**Pull request by cubells**
With these changes, readme can be created without error.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Well, none of the runbot ids should be missing, and the patch should be to add that repo ID. Which one are you talking about?

@sbidoul do you prefer to silently pass this?

**Comment by cubells** [comment]
I detect the problem in two repos: `connector-telephony` and `oca-custom`.

**Comment by pedrobaeza** [comment]
Please propose the patch then on https://github.com/OCA/maintainer-tools/blob/master/tools/repos_with_ids.txt

File: tools/gen_addon_readme.py [hunk 1]
```diff
@@ -141,7 +141,10 @@ def gen_one_addon_readme(repo_name, branch, addon_name, addon_dir, manifest):
                     if fragment[-1] != '\n':
                         fragment += '\n'
                     fragments[fragment_name] = fragment
-    runbot_id = get_runbot_ids()[repo_name]
```
Comment (sbidoul) [inline tools/gen_addon_readme.py]: shorter: `runbot_id = get_runbot_ids()[repo_name].get(repo_name)`

**Comment by sbidoul** [comment]
> do you prefer to silently pass this?

We could support repos without runbot why not, it's a small step towards making this script more generic.

OTOH, it's nice to have a way to detect a missing runbot config.

**Comment by pedrobaeza** [comment]
@cubells can you the changes asked by @sbidoul ?

**Comment by cubells** [comment]
@pedrobaeza @sbidoul done.

**Comment by cubells** [comment]
@pedrobaeza 
A warning is not showed if runbot_id doesn't exist. The reason to do that pr is that user could create readme.rst file **even** if the runbot id doesn't exist: the final readme file will not have the runbot ribbon.
But I can add a warning if you prefer.

**Comment by pedrobaeza** [comment]
Yes, please, see second paragraph of https://github.com/OCA/maintainer-tools/pull/372#issuecomment-421969401

**Comment by cubells** [comment]
@pedrobaeza done in https://github.com/OCA/maintainer-tools/pull/372/files#diff-b5e93c0518459db660ec5afe6d44ad1eR157

**Pull request merged**